Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM alpine:latest
+
+LABEL org.opencontainers.image.authors="Adfinis AG <https://adfinis.com>"
+
+RUN apk add --no-cache \ 
+    bash \
+    jq \
+    curl \
+    tcpdump \
+    wget \
+    openssl \
+    bind-tools 
```

Comment:
Can you be nice and sort these alphabetically?

Reply:
63886a6

Reply:
2d28f41